apps/gauzy/src/app/pages/dashboard/dashboard-switcher/dashboard-switcher.component.html

```
@@ -0,0 +1,129 @@
+<div class="dashboard-switcher">
+	<!-- Chips row: only rendered when at least one custom dashboard exists -->
+	@if (dashboards$ | async; as dashboards) {
+		@if (dashboards.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)